db/models.py

```diff
@@ -14,3 +14,49 @@ class Actor(models.Model):
 
     def __str__(self) -> str:
         return f"{self.first_name} {self.last_name}"
+
+
+class Movie(models.Model):
+    title = models.CharField(max_length=255)
+    description = models.TextField()
+    actors = models.ManyToManyField(
+        Actor,
+        related_name="movies_with_actor"
+    )
+    genres = models.ManyToManyField(
+        Genre,
+        related_name="movies_of_genre"
+    )
+
+    def __str__(self) -> str:
+        return self.title
+
+
+class CinemaHall(models.Model):
+    name = models.CharField(max_length=255)
+    rows = models.IntegerField()
+    seats_in_row = models.IntegerField()
+
+    def __str__(self) -> str:
+        return self.name
+
+    @property
```

Comment:
using the decorator is not necessary. You can remove it

Reply:
But Danylo said in video Calculated Field of this topic, that using decorator property in such cases is an example of good code style and without this decorator 3 tests fail.

Reply:
Yes, it is, you are right